src/Symfony/Component/HttpFoundation/Tests/ResponseFunctionalTest.php

```diff
@@ -23,7 +23,7 @@ public static function setUpBeforeClass(): void
             1 => ['file', '/dev/null', 'w'],
             2 => ['file', '/dev/null', 'w'],
         ];
-        if (!self::$server = @proc_open('exec php -S localhost:8054', $spec, $pipes, __DIR__.'/Fixtures/response-functional')) {
+        if (!self::$server = @proc_open('exec '.\PHP_BINARY.' -S localhost:8054', $spec, $pipes, __DIR__.'/Fixtures/response-functional')) {
```

Comment:
Should we do the same in `src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/AbstractSessionHandlerTest.php`?

Reply:
We should, thank you. Done!